.devcontainer/devcontainer.json

```
@@ -1,35 +1,31 @@
-// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
-// https://github.com/microsoft/vscode-dev-containers/tree/v0.231.6/containers/jekyll
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/jekyll
 {
 	"name": "Jekyll",
-	"build": {
-		"dockerfile": "Dockerfile",
-		"args": {
-			// Update 'VARIANT' to pick a Debian OS version: bullseye, buster
-			// Use bullseye when on local arm64/Apple Silicon.
-			"VARIANT": "bullseye",
-			// Enable Node.js: pick the latest LTS version
-			"NODE_VERSION": "lts/*"
-		}	
-	},
-
-	// Set *default* container specific settings.json values on container create.
-	"settings": {},
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/jekyll:0-bullseye",
```

```suggestion
	"image": "mcr.microsoft.com/devcontainers/jekyll:1-bullseye",
```

Yesterday, we did a release which bumps the major version of this image.
Opened https://github.com/devcontainers/templates/pull/149 just now to sync with those changes.

Ref: https://github.com/devcontainers/images/pull/516 uses ruby 3.2 instead of ruby 2.7 which could be a breaking change for folks.
As this is a major version change, can we ensure that this dev config works well with the website?

---

I bumped the version and it works with ruby 3.2! Thanks for pointing this out.

---

Sweet, thanks!